src/test/java/org/jqassistant/contrib/plugin/hcl/TerraformScannerPluginInfraTest.java

```diff
@@ -0,0 +1,36 @@
+package org.jqassistant.contrib.plugin.hcl;
+
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.jupiter.api.Test;
+
+import com.buschmais.jqassistant.core.scanner.api.DefaultScope;
+import com.buschmais.jqassistant.core.store.api.model.Descriptor;
+import com.buschmais.jqassistant.plugin.common.test.AbstractPluginIT;
+
+public class TerraformScannerPluginInfraTest extends AbstractPluginIT {
+  private static final String FILE_TEST_TF = "/terraform/file.tf";
+
+  @Before
+  public void beginTransaction() {
+    store.beginTransaction();
+  }
+
+  @After
```

Comment:
Rolling back is not needed IMHO, as every test case uses its own in-memory database. Besides this it is sometimes helpful if you can access the database created by a testcase to analyse the generated graph. But this is only my personal opinion.

Reply:
Ok, this is more my personal opinion to do it in that way. It is totaly fine for me, if it works for you.